packages/sirius-web/frontend/sirius-web/src/views/edit-project/EditProjectView.tsx

```diff
@@ -188,42 +196,67 @@ export const EditProjectView = () => {
       />,
     ];
 
+    const nodeTypeRegistry: NodeTypeComponentRegistry = {
+      getNodeStyleContributions: () => {
+        const ellipseNodeStyleContribution: NodeStyleContribution = {
+          name: 'EllipseNodeStyle',
+          fields: `borderColor, borderSize, borderStyle, color`,
```

Comment:
No need for the commas, this is included directly inside the GraphQL query text.